Code/Tools/SceneAPI/SceneData/Behaviors/ImportGroup.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
```

Comment:
This is a strange place to the #pragma once.

It is normally placed beneath the copyright
```suggestion
/*
 * Copyright (c) Contributors to the Open 3D Engine Project.
 * For complete copyright and license terms please see the LICENSE at the root of this distribution.
 *
 * SPDX-License-Identifier: Apache-2.0 OR MIT
 *
 */
 #pragma once
```